Assets/Scripts/Game/Formulas/FormulaHelper.cs

```diff
@@ -275,6 +275,17 @@ public static int CalculateShopliftingChance(PlayerEntity player, int shopQualit
             chance += shopQuality + weightAndNumItems;
             return Mathf.Clamp(chance, 5, 95);
         }
+        
+        // Calculate chance of stealth skill hiding the user.
+        public static int CalculateStealthChance(float distanceToTarget, float globalScale, DaggerfallEntityBehaviour target)
```

Comment:
Are both float params really necessary? It could simply be a single param which is already divided by global scale or the formula could just use the const directly. Either way I don't see the use of this second param for custom formula.

Reply:
True, I went a bit back and forth on this. But decided to take both, since anyone making a mod for stealth in the future might like the ability to change the distance calculation which is done by using those two floats.

Reply:
Yes, okay. My point is that even so the param being supplied is actually a public constant which can be used from any mod code so the param is redundant.

I think it should be removed.

Reply:
Good point. I changed this so it just reads MeshReader.Globalscale dirctly instead.

Reply:
Thank you, I definitely prefer to read the constant. The whole world is constructed on that scale value, it's like a constant of the universe.